app/config.py

```diff
@@ -22,6 +22,10 @@
         'name': 'pending',
         'order': 10,
     },
+    'paused': {
+        'name': 'paused',
+        'order': 10,
```

Comment:
"Global" health is determined using this order. Having pending and paused the same order practically means that one event does not take precedence over the other when determining the global value.  In our setup this would only matter if we had a status board with no components in the `healthy`, `warning` and `failed` categories which is an edge case.

That said I'll update paused value to `15` to be able to distinguish if ever needed and add appropriate `global_messages` to `paused` and `pending`.